### PR TITLE
CJ-78 frontend: configurable base url

### PR DESCRIPTION
**Pull request by johnyka**

### CODE REVIEW

File: asd.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-int add(int a, int b){
-  return a + b;
+int add(int a, int b, int c){
```
Comment (johnyka) [inline asd.txt]: ez szar 💩

**Comment by rdg5** [comment]
Te vagy a :hankey:

File: asd.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-int add(int a, int b){
-  return a + b;
+int add(int a, int b, int k){
+  return a + b + k;
```
Comment (johnyka) [inline asd.txt]: return a + b - k

Comment (EggDice) [reply asd.txt]: I would not do this...

**Pull request merged**